tests/test_utilities.py

```diff
@@ -0,0 +1,95 @@
+from anyway.utilities import is_valid_number, check_is_a_safe_redirect_url
+
+
+# The main logic is implemented in external library, the only reason for this test is to make sure that this library
+# will continue to support israeli numbers
+def test_validate_phone():
+    # Bad numbers
+    short_numbers = [
+        "1",
+        "+213",
+        "+9721234",
+        "972234",
+        "0541243",
+        "+213",
+        "+972-1234",
+        "972-234",
+        "054-1243",
+    ]
+    long_numbers = [
+        "+9721234567890",
+        "031234567890",
+        "0541246805232",
+        "+972-123-456-7890",
+        "03123-4567-890",
+        "05412-4680-5232",
+    ]
+    # Good number
+    good_numbers = [
+        "+9720541234567",
+        "+972541234567",
+        "+97254-123-4567",
+        "+972-54-123-4567",
```

Comment:
`"+972-054-123-4567"` should also be valid, I think - the zero is simply ignored.

Reply:
It should be valid, i will add it to the test.